openbb_terminal/cryptocurrency/cryptocurrency_helpers.py

```diff
@@ -304,6 +304,172 @@ def get_coinpaprika_id(symbol: str):
     return coinpaprika_id
 
 
+def load_from_ccxt(
+    symbol: str,
+    start_date: datetime = (datetime.now() - timedelta(days=1100)),
+    interval: str = "1440",
+    exchange: str = "binance",
+    vs_currency: str = "usdt",
+) -> pd.DataFrame:
+    """Load crypto currency data from CCXT.
```

Comment:
we normally do here like this:

`Load crypto currency data [Source: https://github.com/ccxt/ccxt]`